feature/experimental/security/aaa/kne_tests/tls_authentication_over_grpc_test/tls_authentication_over_grpc_test.go

```diff
@@ -149,15 +152,14 @@ func TestAuthentication(t *testing.T) {
 				}
 			}
 			t.Log("Trying credentials with GNMI Set")
+			json_config, _ := json.Marshal(*deviations.BannerDelimiter + "message of the day" + *deviations.BannerDelimiter)
```

Comment:
don't use underscores in Go names; var json_config should be jsonConfig

[http://golang.org/doc/effective_go.html#mixed-caps](https://www.google.com/url?sa=D&q=http%3A%2F%2Fgolang.org%2Fdoc%2Feffective_go.html%23mixed-caps)

Reply:
Changed to jsonConfig